components/pedigree/resources/src/main/resources/pedigree/view/abstractHoverbox.js

```diff
@@ -267,6 +270,25 @@ define([
                 this.generateHandles();
         },
 
+        setDoNotHideHoverbox: function(raphaelElement) {
+            // raphaelElement is either a set which has .forEach() defined (but has no .node),
+            // or a single element which has .node but not .forEach()
+            if (raphaelElement.node) {
+                raphaelElement.node.setAttribute('class', 'donothidehoverbox ' + raphaelElement.node.getAttribute('class'));
+                // a hack for raphael <tspan> complication when using SVG (on most modern browsers)
+                if (raphaelElement.node.children &&
+                    raphaelElement.node.children.length > 0 &&
+                    raphaelElement.node.children[0].nodeName == "tspan") {
+                    raphaelElement.node.children[0].setAttribute("class","donothidehoverbox");
+                }
+            } else {
+                var _this = this;
+                raphaelElement.forEach(function(element) {
+                    _this.setDoNotHideHoverbox(element);
+                });
+            }
+        },
```

Comment:
This function worries me a little. There are A LOT of elements with this class name in a pedigree (e.g. 480 in a family with 21 members). Additionally, some class attributes look weird, see for example:

![image](https://user-images.githubusercontent.com/651980/29382256-fb710c90-8299-11e7-9edc-ca3d2d9c6162.png)